metadata.php

```diff
@@ -30,7 +30,8 @@
     'email'             => 'support.at@wirecard.com',
     'extend'            => array (
         \OxidEsales\Eshop\Application\Model\Order::class          => \Wirecard\Oxid\Extend\Order::class,
-        \OxidEsales\Eshop\Application\Model\PaymentGateway::class => \Wirecard\Oxid\Extend\Payment_Gateway::class
+        \OxidEsales\Eshop\Application\Model\PaymentGateway::class => \Wirecard\Oxid\Extend\Payment_Gateway::class,
+        \OxidEsales\Eshop\Application\Controller\Admin\PaymentMain::class => \Wirecard\Oxid\Extend\Payment_Main::class
```

Comment:
I started in WDPD-25 to use the same base path for the extended classes as the class they extend from. In this case that would be `\Wirecard\Oxid\Extend\Controller\Admin\Payment_Main::class`